DeBERTa/deberta/ops.py

```diff
@@ -40,29 +42,29 @@ class XSoftmax(torch.autograd.Function):
   """
 
   @staticmethod
-  def forward(self, input, mask, dim):
+  def forward(cts, input, mask, dim):
```

Comment:
Are those changes necessary?

Reply:
Not necessary. But it's weird to use self in a static method. We use "cts" in the BigBird model. That's why I change it.